src/transformers/models/bloom/modeling_bloom.py

```diff
@@ -693,7 +693,7 @@ def forward(
             past_key_values = DynamicCache.from_legacy_cache(past_key_values)
             logger.warning_once(
                 "Using `past_key_values` as a tuple is deprecated and will be removed in v4.45. "
-                "Please use an appropriate `Cache` class (https://huggingface.co/docs/transformers/v4.41.3/en/internal/generation_utils#transformers.Cache)"
+                "Please use an appropriate `Cache` class (https://huggingface.co/docs/transformers/internal/generation_utils#transformers.Cache)"
```

Comment:
found in my quick scan for deprecations (CMD+F on "v4.41") -- let's use our latest docs to guide our users